### DISCUSSION TITLE
Please add a redirect from /pull to /pulls

### DISCUSSION THREAD

**Discussion by carltongibson** [community/community#5021] [Pull Requests] [answered]
For years: 

* Browsing PR at `<owner>/<repo>/pull/1234`
* Want to go to list of PRs. 
* Adjust URL quickly, but forget the `s` `<owner>/<repo>/pull`
* `404`. 😬
* Sigh. Add `s`. 

A redirect to `pulls` from `pull` would save a lot of annoyance. The way browsers select path segments when editing URLs makes this happen a lot. 

Thanks.

**Comment by simonw** [comment]
I've been caught out by this one a bunch of times too - especially since GitHub's URL design in general is so clean and consistent and hackable.

**Comment by tim-schilling** [comment]
~~To anyone reading this that hasn't used the upvote button, it's the little arrow above the number to the left of the title. Click that.~~ The UI changed for the better.

**Comment by jezdez** [comment]
YES PLEASE!

**Comment by drewbrew** [comment]
Corollary: if I'm at `/pulls` and edit the URL to be `/pulls/42`, I'd like the site to redirect me in the reverse direction

**Reply by drakored** [reply]
This.  One of the best things about GH is how semantically clean their endpoints are.  This would give me an OCD twitch if they implemented it the other way.

**Comment by sethmlarson** [comment]
This happens to me on a near-weekly basis!

**Comment by dipree** [comment, marked as answer]
Hey, just letting you know that `github.com/owner/repository/pull` now redirects to `github.com/owner/repository/pulls`.

**Reply by carltongibson** [reply]
Wow, thanks. That’s super!

**Reply by graingert** [reply]
It seems @owner doesn't have a repository called repository

**Reply by jezdez** [reply]
Amazing, thanks @dipree et al!

**Reply by ajaxsys** [reply]
@dipree 
Still not work now?

for example:
https://github.com/tiimgreen/github-cheat-sheet/pull/15.diff
is OK

https://github.com/tiimgreen/github-cheat-sheet/pulls/15.diff
return 404 (with "s")

As intelliJ IDE only looks into `pulls`, so it will go wrong in intellij

**Reply by dipree** [reply]
@ajaxsys the forward is from `github.com/owner/repository/pull` to `github.com/owner/repository/pulls` to avoid typos but does not resolve deep links. I'm not familiar with IntelliJ unfortunately but I'd be curious what they say. Have you created a ticket there?

**Reply by yurri-yeskov** [reply]
Thanks

**Reply by BlueSkyClouds** [reply]
thanks

**Reply by Asoomv** [reply]
> Wow, thanks. That’s super!
